src/Lime.Protocol/Network/ChannelCommandProcessor.cs

```diff
@@ -21,6 +21,11 @@ public ChannelCommandProcessor()
         
         public async Task<Command> ProcessCommandAsync(ICommandSenderChannel commandSenderChannel, Command requestCommand, CancellationToken cancellationToken)
         {
+            if (cancellationToken.IsCancellationRequested)
```

Comment:
You can write just:
`cancellationToken.ThrowIfCancellationRequested()`

Reply:
Done :)